runtime/onert/core/src/exec/train/TrainableExecutor.cc

```diff
@@ -187,6 +187,10 @@ void TrainableExecutor::backwardImpl(const ExecutionObservee &subject, uint32_t
     for (auto &&index : _backward_order)
     {
       const auto &code = _code_map.at(index);
+      if (!code.op->isRequiredForBackward())
```

Comment:
It's a equivalent of https://github.com/Samsung/ONE/pull/13199/files#diff-c3aea1c2d60b52608ea27f07b39c07268fd1e4d9fbc88fe6c3517318fa713cf3R167-R170 but was missing in this path